src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporter.cs

```diff
@@ -75,6 +88,9 @@ public override ExportResult Export(in Batch<LogRecord> logRecordBatch)
         catch (Exception ex)
         {
             OpenTelemetryProtocolExporterEventSource.Log.ExportMethodException(ex);
+
+            OpenTelemetryProtocolExporterEvents.ExportMethodException(this.openTelemetryEventLogger, ex.ToInvariantString());
```

Comment:
we just need to pass exception object itself to logger. It won't get serialized, unless someone is listening to that. If someone is listening, the providers know how to serialize exception into string representation, where EventSource required raw types like string, and can't work with types like `Exception`